config/locales/date.sv.yml

```diff
@@ -0,0 +1,4 @@
+sv:
+  date:
+    month_names: ['Januari', 'Februari', 'Mars', 'April', 'Maj', 'Juni', 'Juli',
+                  'Augusti', 'September', 'Oktober', 'November', 'December']
```

Comment:
Why do we need this? The `date:month_names` key is also in sv.yml

Reply:
You're right - I missed that.  (which might be an argument for breaking up our huge locale files into domain-specific files - but that would be for another story, of course).

Reply:
I added a chore for this in PT: https://www.pivotaltracker.com/story/show/152857819